apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -0,0 +1,5 @@
+export function getManifest(appType, locale) {
```

Comment:
two small things that could be future / clean-up work:
1. does any `appType` work here? if it only works for gamelab/spritelab, we may want to validate this
2. super nit: we could default this `locale` param to `en_us`

Reply:
you just pushed a commit that already does 1, so ignore that!